erc-draft-compressed-rpc-link-format.md

```diff
@@ -0,0 +1,1224 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 8050
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.